code/modules/mob/living/living.dm

```diff
@@ -493,8 +493,25 @@
 		s_active.close(src)
 
 	if(lying && !buckled && prob(getBruteLoss()*200/maxHealth))
-
 		makeTrail(newloc, T, old_direction)
+	
+	if(bleed_rate && !buckled && has_gravity(loc) && lying)
+		if(prob(2) && istype(loc, /obj/structure/closet))
+			visible_message("<span class='warning'>[src] bumps their head against \the [src]!</span>")
+			apply_damage(rand(1, 2.5), BRUTE, "head")
+		else
+			if(prob(getBruteLoss() / 5) && health > HEALTH_THRESHOLD_CRIT)
+				bleed(rand(bleed_rate*0.75, bleed_rate))
+				add_splatter_floor(get_turf(src))
+				visible_message("<span class='warning'>\The [src] loses some blood from being dragged!</span>")
```

Comment:
The message here is a little weird. Perhaps something like "[src]'s wounds are torn wider from being dragged across the floor!" would work better?